src/main/java/pl/coderslab/projectwybankuj/model/UploadLogoResponse.java

```diff
@@ -0,0 +1,13 @@
+package pl.coderslab.projectwybankuj.model;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class UploadLogoResponse {
+    private String logoName;
+//    private String fileDownloadUri;
+    private String fileType;
+
+    public UploadLogoResponse(String logoName, String fileDownloadUri, String fileType, long size) {
```

Comment:
Skoro jest AllArgsConstructor to można ten usunąć ale radizłbym się naraize powstrzymac od Lomboka